### ISSUE TITLE
Improve this so the repositories do not contain all the data

### ISSUE THREAD

**Issue by mwestphal** [lorensen/VTKExamples#231]
This repository should be improved to reduce it's size (currently over 1Gb)
 * Data and Baseline could be stored in VTK data and baselines with on demand download
 * .html file could be stored as a secondary submodule

This would allow this repository integration into VTK so the examples could be considered actual tests and more easy to maintain for developers.

Another solution would be to put all the source files directly in VTK and this repository would use them to build the website.

**Comment by lorensen** [comment]
When you clone it, set the depth to 1. I will look at better organization.

**Comment by banesullivan** [comment]
Pehaps we could collaborate here - for [`vtki`](https://github.com/vtkiorg/vtki), we host an examples data repository over at: https://github.com/vtkiorg/vtk-data - this holds data files only

There are so many wonderful examples here and I'd love to showcase how `vtki` simplifies some of these examples - perhaps I could migrate the data here over to that repo and help implement a data downloading routine like what we have for `vtki` so users can download example data on the fly:

```py
from vtki import examples
# Load St Helens DEM and warp the topography
mesh = examples.download_st_helens().warp_by_scalar()
mesh.plot(opacity='linear')
```

or perhaps simply submodule the data repository so the CI's can quickly download the data files when building the examples?

**Comment by lorensen** [comment]
We use GitHub lfs for large data. Part of problem is large history. I'll
revisit this. I really don't want to rely on another trip.

Thanks

**Comment by banesullivan** [comment]
Fair enough - considering this repo is under an Apache license, I'll merge the data folder here into our example data repo as an independent effort. 

I'm totally willing to add you, @lorensen, to @vtkiorg as an admin on the data repo if you'd like to collaborate moving forward.
